corehq/apps/hqadmin/models.py

```diff
@@ -19,7 +19,7 @@ class SQLHqDeploy(models.Model):
     diff_url = models.CharField(max_length=126, null=True)
     couch_id = models.CharField(max_length=126, null=True, db_index=True)
 
-    class Meta:
+    class Meta(object):
```

Comment:
Curious why this was required? I thought it wasn't in Python 3

Reply:
It isn't required, just an incidental change made while troubleshooting.